### DISCUSSION TITLE
Have all bioinformaticians installed bioconda？

### DISCUSSION THREAD

**Discussion by whatabear** [community/community#39974] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Anyone could give me some suggestions? seriously...

**Comment by radar-blip** [comment]
About what?

**Reply by whatabear** [reply]
I am a new in bioinfo, i used to install the bio-softwares by GitHub or official websites, but i seen a lot discussions about bioconda installation,is it really need?
